falkordb-node/node-entrypoint.sh

```diff
@@ -409,45 +409,46 @@ if [ "$RUN_NODE" -eq "1" ]; then
 fi
 
 if [ "$RUN_SENTINEL" -eq "1" ]; then
-  sed -i "s/\$ADMIN_PASSWORD/$ADMIN_PASSWORD/g" $SENTINEL_CONF_FILE
-  sed -i "s/\$FALKORDB_USER/$FALKORDB_USER/g" $SENTINEL_CONF_FILE
-  sed -i "s/\$FALKORDB_PASSWORD/$FALKORDB_PASSWORD/g" $SENTINEL_CONF_FILE
-  sed -i "s/\$LOG_LEVEL/$LOG_LEVEL/g" $SENTINEL_CONF_FILE
-
-  # When LB is in place, change external dns to internal ip
-  sed -i "s/\$SENTINEL_HOST/$NODE_EXTERNAL_DNS/g" $SENTINEL_CONF_FILE
-
-  echo "Starting Sentinel"
-
-  if [[ $TLS == "true" ]]; then
-    echo "port 0" >>$SENTINEL_CONF_FILE
-    echo "tls-port $SENTINEL_PORT" >>$SENTINEL_CONF_FILE
-    echo "tls-cert-file $TLS_MOUNT_PATH/tls.crt" >>$SENTINEL_CONF_FILE
-    echo "tls-key-file $TLS_MOUNT_PATH/tls.key" >>$SENTINEL_CONF_FILE
-    echo "tls-ca-cert-file $ROOT_CA_PATH" >>$SENTINEL_CONF_FILE
-    echo "tls-replication yes" >>$SENTINEL_CONF_FILE
-    echo "tls-auth-clients no" >>$SENTINEL_CONF_FILE
-  else
-    echo "port $SENTINEL_PORT" >>$SENTINEL_CONF_FILE
-  fi
-
-  redis-server $SENTINEL_CONF_FILE --sentinel --logfile $SENTINEL_LOG_FILE_PATH &
-  sentinel_pid=$!
-  tail -F $SENTINEL_LOG_FILE_PATH &
-
-  sleep 10
+  if [[ "$NODE_INDEX" == "0" || "$NODE_INDEX" == "1" ]];then
```

Comment:
If all the code inside the first condition is also inside the second condition, why don't you merge them and clean the code?